e2e/Runner.js

```diff
@@ -18,10 +18,16 @@ module.exports = class Runner {
         });
     }
 
-    static startApp(root) {
+    static startApp(root, isMockApp) {
```

Comment:
you can avoid this parameter by inspective the type of the first parameter, _.isString()